### ISSUE TITLE
fails to run tests on Eclipse Oxygen

### ISSUE THREAD

**Issue by yilativs** [pitest/pitclipse#72]
Fails to run tests on Eclipse Oxygen.
Execution throws exception to console:

```
Closing server
Closed
Exception in thread "main" java.lang.NoSuchMethodError: joptsimple.OptionParser.acceptsAll(Ljava/util/List;Ljava/lang/String;)Ljoptsimple/OptionSpecBuilder;
	at org.pitest.mutationtest.commandline.OptionsParser.<init>(OptionsParser.java:122)
	at org.pitest.mutationtest.commandline.MutationCoverageReport.main(MutationCoverageReport.java:36)
	at org.pitest.pitclipse.pitrunner.PitRunner.runPIT(PitRunner.java:46)
	at org.pitest.pitclipse.pitrunner.PitRunner.main(PitRunner.java:25)
```

**Comment by velo** [comment]
For me it's failing a well, but it only prints

```
Error: Could not find or load main class org.pitest.pitclipse.pitrunner.PitRunner
```

**Comment by benhjt** [comment]
I can't even get it to install properly. It installs without any errors/warnings but then does not show up in any context menu to run a test class as a PIT mutation test.

**Comment by velo** [comment]
FWIW, i build it locally to get it working

**Comment by jenildesai25** [comment]
Can you tell me how to build it locally

**Comment by davidselvaraj** [comment]
I am also getting the same error.  I am using STS 3.9.2 and getting the same error as the original poster.
```
Exception in thread "main" java.lang.NoSuchMethodError: joptsimple.OptionParser.acceptsAll(Ljava/util/List;Ljava/lang/String;)Ljoptsimple/OptionSpecBuilder;
	at org.pitest.mutationtest.commandline.OptionsParser.<init>(OptionsParser.java:122)
	at org.pitest.mutationtest.commandline.MutationCoverageReport.main(MutationCoverageReport.java:36)
	at org.pitest.pitclipse.pitrunner.PitRunner.runPIT(PitRunner.java:46)
	at org.pitest.pitclipse.pitrunner.PitRunner.main(PitRunner.java:25)
```

**Comment by echebbi** [comment]
Fixed by #80. Will be included in the next release.

**Comment by echebbi** [comment]
The new `2.0.0` release is finally here and should fix the issue: https://github.com/pitest/pitclipse/releases/tag/v2.0.0
